### PR TITLE
Added the challenge solution.

### PR DESCRIPTION
**Pull request by Estrax**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -58,39 +58,94 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+// let car = inventory.filter((elem) => elem.id==33)[0];
+
+console.log(`Car 33 is a ${car.car_year} ${car.car_make} ${car.car_model}` );
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+function challenge2(arr){
```
Comment (BrandonVilaychith) [inline assignments/arrays.js:83]: The last car they were looking for is {"id":50,"car_make":"Lincoln","car_model":"Town Car","car_year":1999} based on the array. Your solution for challenge 2 returns the wrong one

File: assignments/arrays.js [hunk 2]
```diff
@@ -58,39 +58,94 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+console.log(lastCar.car_make, lastCar.car_model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+
+// typical sort:
+// let carModels = inventory.sort((a, b) => (a.car_model > b.car_model)? 1 : (b.car_model > a.car_model) ? -1 : 0);
+let carModels = inventory.sort((a, b) => a.car_model > b.car_model);
```
Comment (BrandonVilaychith) [inline assignments/arrays.js:101]: The car models are not listed in alphabetical order, I would try to use an if statement in the sort method to get it to sort correctly

**Review by BrandonVilaychith — COMMENTED** [review]
Great job on the project, there are some fixes to be made in the array challenge but besides that good work, I how you used classes in the objects portion

// Project Rating
2 (considered a 3 once arrays.js is fixed)

File: assignments/arrays.js (hunk 1 above)
Comment (Estrax) [reply assignments/arrays.js:83]: Aww, I misunderstood it somehow & overcomplicated this one as well - don't know why, but I thought it's all about the newest car out there. Fixed in #2.

```js
function challenge2(arr){
    return arr[arr.length-1];
}
```

File: assignments/arrays.js (hunk 2 above)
Comment (Estrax) [reply assignments/arrays.js:101]: Exactly - so I tuned a little bit the one I commented above and fixed the letter-case issue.

Fixed in #2.

**Pull request merged**